usecases/interface.go

```diff
@@ -35,3 +40,24 @@ type (
 		Execute(service entities.ServiceName) (entities.Deployment, error)
 	}
```

Comment:
should these interface definition be unnamed in the same way as feature added below?